main.go

```diff
@@ -26,29 +26,33 @@ func main() {
 		c.File("static/index.html")
 	})
 
+	router.GET("/task/", func(c *gin.Context){
+		err, allTasks := service.GetAllTasks()
+		if err != nil {
+			c.JSON(http.StatusBadRequest, "Could not get all tasks");
+			return
+		}
+		c.JSON(http.StatusOK, allTasks)
 
-	testEntry := model.Task{Description:"My Test Time Entry Here."}
-	anotherTestEntry := model.Task{Description:"Just messing with the project structure"}
-
-	log.Print("Hello ",  testEntry.Description)
-	log.Print("Hello Again and more. ",  anotherTestEntry.Description)
+	})
 
-	// example on how to serve json...
 	router.GET("/task/:id", func(c *gin.Context){
 		id :=  c.Param("id")
 		i, err := strconv.Atoi(id)
 
 		if err != nil { //FIXME there must be a better way to validate the parameter, that it is an int.
 			c.JSON(http.StatusBadRequest, "id must be a number");
```

Comment:
Could we have some helper method to handle getting required params with type?

Reply:
yes. most likely there is a "by the book" way of doing it in gin... just have to figure out what it is.. :)